src/sendalert.cpp

```diff
@@ -72,15 +72,17 @@ void ThreadSendAlert()
     CAlert alert;
     alert.nRelayUntil   = GetTime() + 15 * 60;
     alert.nExpiration   = GetTime() + 10 * 365 * 24 * 60 * 60;
-    alert.nID           = 1006;  // use https://github.com/zcash/zcash/wiki/specification#assigned-numbers to keep track of alert IDs
-    alert.nCancel       = 1005;  // cancels previous messages up to this ID number
+    alert.nID           = 1007;  // use https://github.com/zcash/zcash/wiki/specification#assigned-numbers to keep track of alert IDs
+    alert.nCancel       = 1006;  // cancels previous messages up to this ID number
 
     // These versions are protocol versions
     // 170002 : 1.0.0
     // 170006 : 1.1.2
     // 170007 : 2.0.0
+    // 170010 : 2.1.2
```

Comment:
I thought this was changed in 2.1.0?

Reply:
No, 2.1.2 set testnet activation for Heartwood: #4469